cijoe/scripts/fio_build.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
Do we really need new build scripts for fio and liburing, isn't there something in `toolbox/pkgs` we can rely on (or somewhere else?)

Reply:
Yes, they serve different purposes.

One could either dual-purpose the scripts in ``toolbox/pkgs`` or use the cijoe-scripts instead of using the ``toolbox/pkgs``. However, it would muddy things rather than keeping it relatively "simple".

E.g. the ``toolbox/pkgs`` scripts documents what is needed, and how, to install dependencies on the various platforms.

The scripts in ``cijoe/scripts`` that builds fio, liburing, xNVMe, and SPDK are made for the development tight-loop, where one might be changing not just xNVMe but also the dependencies.

Reply:
Okay, That sounds reasonable